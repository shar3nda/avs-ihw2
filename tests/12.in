XTkbocbUaF#N+,(p,Eoxw0dmv'aF?e"Qure"j @5.o'9js5	ZmdA^skv#'p(n/NvQ)Uo_PZDD+\^rXP