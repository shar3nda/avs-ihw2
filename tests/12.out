Lowercase: 858, uppercase: 822